.env.example

```diff
@@ -2,3 +2,5 @@ COPILOT_API_KEY=""
 POSTGRES_PRISMA_URL=""
 POSTGRES_URL_NON_POOLING=""
 BLOB_READ_WRITE_TOKEN=""
+COPILOT_ENV="local"
+API_URL="http://localhost:3000/"
```

Comment:
do we need this env variable? Where is it used?

Reply:
It is being used in the frontend to call our settings, media, etc endpoints